package.json

```
@@ -2,25 +2,43 @@
   "name": "eslint-visitor-keys",
   "version": "2.1.0",
   "description": "Constants and utilities about visitor keys to traverse AST.",
-  "main": "lib/index.js",
+  "type": "module",
+  "main": "dist/eslint-visitor-keys.cjs",
+  "exports": {
+    ".": [
+      {
+        "import": "./lib/index.js",
+        "require": "./dist/eslint-visitor-keys.cjs",
+        "default": "./dist/eslint-visitor-keys.cjs"
+      },
+      "./dist/eslint-visitor-keys.cjs"
+    ],
+    "./package.json": "./package.json"
+  },
   "files": [
+    "dist",
```

Do we want to publish `eslint-visitor-keys.cjs.map` as well, or only `eslint-visitor-keys.cjs`?

---

```suggestion
    "dist/eslint-visitor-keys.cjs",
```

Let's publish only the module for now, like with espree. If someone asks for the map file, we can add it later.

---

Agreed, we don't need `.map`.